less/v2/npc.less

```diff
@@ -549,6 +549,40 @@
           }
         }
       }
+
+      /* Spells */
+      .spellcasting.card {
+        input {
+          background: var(--dnd5e-background-5);
+          border: 1px solid transparent;
+          transition: all 250ms ease;
+          text-align: center;
+          font-weight: bold;
+          font-family: var(--dnd5e-font-roboto-slab);
+          font-size: var(--font-size-16);
+          height: 22px;
+
+          &:hover, &:focus {
+            border: 1px solid var(--dnd5e-color-gold);
+            box-shadow: 0 0 6px var(--dnd5e-color-gold);
+          }
+        }
+
+        select {
+          border: none;
+          background: none;
+
+          option { color: var(--color-text-dark-primary); }
+        }
+
+        .config-button {
+          flex: 1;
+          display: grid;
+          place-content: center;
+        }
+
+        .spellcasting-ability { color: var(--dnd5e-color-gold); }
```

Comment:
Very difficult to read.

<img width="199" alt="Screenshot 2024-07-08 at 15 20 28" src="https://github.com/foundryvtt/dnd5e/assets/19979839/340cc9ed-c512-44b9-b41c-eb64c995b0b5">

Reply:
Looks ok to me, but I have added a text shadow to hopefully make it a bit clearer.